pom.xml

```diff
@@ -65,6 +65,10 @@
     <!-- Whether the build should fail if findbugs finds any error. -->
     <!-- It is strongly encouraged to leave it as true. Use false with care only in transient situations. -->
     <findbugs.failOnError>true</findbugs.failOnError>
+    <!-- Defines a default FindBugs threshold. -->
+    <!-- In Plugin POM 2.x the default value was Medium, but it was not enough to discover some null handling cases -->
+    <findbugs.threshold>Low</findbugs.threshold>
```

Comment:
I'm OK with the idea, a bit more unsure about the ratio of false-positives this could detect.
Currently, I like the fact I'm not bothered that often, and when I do it's almost always a good catch. I wouldn't like to see dozens of errors with only 1 actual in the middle.